docs/README

```diff
@@ -30,6 +30,9 @@ from this dir::
 
 Documentation will be generated (in HTML format) inside the ``build/html`` dir.
 
+If you get the error "ImportError: No module named sphinx_rtd_theme" run:
+    sudo pip install   sphinx-rtd-theme
```

Comment:
I think this is not need, because error message is pretty clear states that module is missing.